llvm/lib/CodeGen/SelectionDAG/SelectionDAGISel.cpp

```diff
@@ -180,6 +182,21 @@ static const bool ViewDAGCombine1 = false, ViewLegalizeTypesDAGs = false,
                   ViewSchedDAGs = false, ViewSUnitDAGs = false;
 #endif
 
+#ifndef NDEBUG
+#define ISEL_DUMP(X)                                                           \
+  do {                                                                         \
+    if (llvm::DebugFlag &&                                                     \
+        (isCurrentDebugType(DEBUG_TYPE) ||                                     \
+         (isCurrentDebugType(ISEL_DUMP_DEBUG_TYPE) && MatchFilterFuncName))) { \
+      X;                                                                       \
+    }                                                                          \
+  } while (false)
+#else
+#define ISEL_DUMP(X)                                                           \
```

Comment:
Just put the whole macro on one line like Debug.h

`#define ISEL_DUMP(X) do { } while (false)`

Reply:
clang-format somehow disagrees with this.

Reply:
Well, since we don't need to pass the formatting check to commit this change, I'll just make it one-line.